ibm/resource_ibm_private_dns_custom_resolver.go

```diff
@@ -155,11 +154,11 @@ func resouceIBMPrivateDNSCustomResolverCreate(context context.Context, d *schema
 		crDescription = des.(string)
 	}
 
-	crLocations := d.Get(pdnsCustomResolverLocations).(*schema.Set)
+	// crLocations := d.Get(pdnsCustomResolverLocations).(*schema.Set)
```

Comment:
location field is made optional..
that means user can stll give location.. please use d.getok() and then set values in customResolverOption